packages/augur-sdk/src/state/create-api.ts

```diff
@@ -125,11 +125,11 @@ export async function createClient(
     connector,
     zeroX,
     enableFlexSearch
-  );{}
+  );
 
   // Delay loading of the browser mesh until we're finished syncing
   client.events.once(SubscriptionEventName.BulkSyncComplete, () => {
-    if (config.zeroX?.mesh?.enabled && createBrowserMesh) {
+    if (!config?.sdk?.enabled && config.zeroX?.mesh?.enabled && createBrowserMesh) {
```

Comment:
We don't want to run the browser mesh if we're using the sdk server. This doesn't actually trigger as things stand because the BulkSyncComplete event doesn't get emitted when using the sdk server... but if that changes then this will prevent an error.